coderd/httpapi/name.go

```diff
@@ -79,3 +79,16 @@ func TemplateDisplayNameValid(str string) error {
 	}
 	return nil
 }
+
+// UserRealNameValid returns whether the input string is a valid real user name.
+func UserRealNameValid(str string) error {
+	if len(str) > 64 {
+		return xerrors.New("must be <= 64 characters")
+	}
```

Comment:
This is going to annoy a lot of people! We can probably afford a higher limit here.

Reply:
Oh, I assumed that 64 is too much, but I can increase to 128.

Reply:
Done